source/run-command/run-command.rst

```diff
@@ -129,6 +131,10 @@ The logical session ID MUST be included under ``lsid`` in the command sent to th
 
 * See Driver Sessions' section on `Sending the session ID to the server on all commands <https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst#sending-the-session-id-to-the-server-on-all-commands>`_
 
+The command sent to the server MUST gossip the ``$clusterTime`` if cluster time support is detected.
+
+* See Driver Sessions' section on `Gossipping the cluster time <https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst#gossipping-the-cluster-time>`_
```

Comment:
Is this only relevant to `RunCursorCommand`, or was it something missed in [DRIVERS-2577](https://jira.mongodb.org/browse/DRIVERS-2577)?

Reply:
Yep it was missed, my goal was to make note of all the "edits" the driver makes to the command on behalf of the user. Do all servers past a certain version send `$clusterTime`? We can add to the runCommand.yml tests a confirmation that it gets attached 🤔

Reply:
[Gossipping the cluster time](https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst#gossipping-the-cluster-time) in the session spec is probably the relevant docs for that. It talks about cluster time applying to wire version 6+ (3.6+), but I didn't see any mention of topology requirements at a quick glance (and don't recall offhand).